.travis.yml

```diff
@@ -6,30 +6,81 @@
 # http://www.eclipse.org/legal/epl-v20.html
 #
 # SPDX-License-Identifier: EPL-2.0
-sudo: false
+sudo: true
 language: java
-cache:
-  apt: true
-  directories:
-    - $HOME/.m2
-before_cache:
-  - rm -rf $HOME/.m2/repository/org/eclipse/sw360/antenna/
-jdk:
-  - openjdk8
-  # - openjdk10
-  # - openjdk11
-  # - oraclejdk8
-  # - oraclejdk9
-  # - oraclejdk11
+addons:
+  apt:
+    packages:
+      - git
+# cache:
```

Comment:
Should we enable caching again?

Reply:
I would enable caching in the next PR, I am also unsure whether it helps. Travis says: if you cache something which you would load from the internet, then loading it from the internet might be faster than loading it from the travis cache.